ppcs/ppcTODO-metaoperator-flags.md

```diff
@@ -0,0 +1,166 @@
+# Meta-operator Flags to Modify Behaviour of Operators
+
+## Preamble
+
+    Author:  Paul Evans <PEVANS>
+    Sponsor:
+    ID:      TODO
+    Status:  Exploratory
+
+## Abstract
+
+Defines a new syntax for adding behaviour-modifying flags to operators, in order to extend the available set of behaviours.
+
+## Motivation
+
+This idea came out of discussions around the proposal to add new `equ` and `===` operators, which are aware of the special nature of the `undef` value. The idea is that rather than adding just two new special-purpose operators, a far more flexible and extensible idea may be to allow syntax for putting flags on the existing operators.
+
+The central idea here is that existing operators can be modified by a set of single-letter suffix flags, that could be considered slightly similar to the concept of regexp pattern flags.
+
+## Rationale
+
+(explain why the (following) proposed solution will solve it)
+
+## Specification
+
+Following the name of a regular value-returning infix operator, an optional set of flag letters can be supplied after a separating colon:
+
+```perl
+$x eq:u $y
```

Comment:
I added prospective support for highlighting this to tree-sitter-perl; looks pretty solid, actually
![image](https://github.com/user-attachments/assets/713ef235-f992-43eb-9d75-d37ed16735dc)